EIPS/eip-7092.md

```diff
@@ -0,0 +1,342 @@
+---
+eip: 7092
+title: Financial Bonds
+description: This interface defines a specification for financial bonds tokenization
```

Comment:
This description restates the title too much. Try to add new information in the description that isn't in the title.